README.md

```diff
@@ -85,932 +119,667 @@ Unlike many authorization systems where a service controls access to resources i
 └─────────────┘   └─────────────┘   └─────────────┘
 ```
 
-# 2. Terminology
+This additionally allows UCAN to model auth for [eventually consistent and replicated state][overcoming SSI].
 
-## 2.1 Roles
+# Roles
```

Comment:
```suggestion
## Roles
```
I think single-hash renames the file in some forms of markdown? in any case my markdown linter plugin discourages using them within a document